Kafka/KafkaJCAAPI/src/main/java/fish/payara/cloud/connectors/kafka/inbound/KafkaAsynchWorker.java

```diff
@@ -110,7 +112,12 @@ public KafkaAsynchWorker(EndpointKey key, WorkManager wm) {
         }
 
         consumer = new KafkaConsumer(key.getSpec().getConsumerProperties());
-        consumer.subscribe(Arrays.asList(key.getSpec().getTopics().split(",")));
+        final Pattern topicsPattern = createTopicsPattern(key);
```

Comment:
Does it have to be either or? Or could we subscribe by topic names and pattern if both exist?

Reply:
Good point. I actually assumed that it was one or the other seen from the KafkaConsumer point of view. Will test out if using both at the same time could work.

Reply:
Specifying both topic list and pattern is not allowed:

java.lang.IllegalStateException: Subscription to topics, partitions and pattern are mutually exclusive
	at org.apache.kafka.clients.consumer.internals.SubscriptionState.setSubscriptionType(SubscriptionState.java:162)